src/mumble/PulseAudio.cpp

```diff
@@ -775,7 +775,7 @@ void PulseAudioSystem::stream_restore_read_callback(pa_context *c, const pa_ext_
 	}
 }
 
-void PulseAudioSystem::restore_volume_success_callback(pa_context *c, int success, void *userdata) {
+void PulseAudioSystem::restore_volume_success_callback(pa_context * /*c*/, int /*success*/, void *userdata) {
```

Comment:
Mew. That's a way I don't like. Why add extra lines into your function body for a variable you won't use in the whole function. Commenting it out (or removing the name) makes it obvious it is actually unused. Q_UNUSED just tells me that it maybe was at some point unused and could possibly still be.